compass/landice/tests/antarctic/antarctic.cfg

```diff
@@ -0,0 +1,29 @@
+# config options for antarctic test cases
+[antarctic]
+
+# number of levels in the mesh
+levels = 10
+
+# distance from ice margin to cull (km).
+# Set to a value <= 0 if you do not want
+# to cull based on distance from margin.
+cull_distance = 5.0
+
+# mesh density parameters
+# minimum cell spacing (meters)
+min_spac = 1.e4
+# maximum cell spacing (meters)
+max_spac = 1.e5
```

Comment:
These are presumably for testing but they're 10 times coarser than the mesh name suggests.

Reply:
Thanks for pointing it out, @xylar. That's right that it these coarse min/max cell spacing were defined for a quick testing, and I didn't feel the need to change the output filename that is hard-coded in the main code `mesh.py`. However, I've switched the values back to `min_spac = 1.e3` and `max_spac = 1.e4` in this file to be consistent within this PR still.

Reply:
I still think it's worth discussing the file name a bit further. If I were a user and I changed the resolution, I would be mildly annoyed that the file name wasn't right.